packages/cli/src/commands/init/banner.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+import chalk from 'chalk';
+
+const reactLogoArray = [
+  '                                                                ',
+  '                                                                ',
+  '                                                                ',
+  '                  ######                ######                  ',
+  '                ###     ####        ####     ###                ',
+  '               ##          ###    ###          ##               ',
+  '               ##             ####             ##               ',
+  '               ##             ####             ##               ',
+  '               ##           ##    ##           ##               ',
+  '               ##         ###      ###         ##               ',
+  '                ##  ########################  ##                ',
+  '             ######    ###            ###    ######             ',
+  '         ###     ##    ##              ##    ##     ###         ',
+  '      ###         ## ###      ####      ### ##         ###      ',
+  '     ##           ####      ########      ####           ##     ',
+  '    ##             ###     ##########     ###             ##    ',
```

Comment:
btw, have you tried if it looks good with 2 spaces padding instead of 4? (similar for the top, I think 1 or 2 is enough)